File: beautify_app/lib/Models/TenanlModels/CreateTenantDto.dart
class CreateTenantDto {
  late String? tenancyName;
  late String? name;
  late String? adminEmailAddress;
  late String? connectionString;
  late String? password;
  late bool? isActive;

  CreateTenantDto(
      {this.tenancyName,
      this.name,
      this.adminEmailAddress,
      this.connectionString,
      this.password,
      this.isActive});

  CreateTenantDto.fromJson(Map<String, dynamic> json) {
    tenancyName = json['tenancyName'];
    name = json['name'];
    adminEmailAddress = json['adminEmailAddress'];
    connectionString = json['connectionString'];
    password = json['password'];
    isActive = json['isActive'];
  }

  Map<String, dynamic> toJson() {
    final Map<String, dynamic> data = <String, dynamic>{};
    data['tenancyName'] = tenancyName;
    data['name'] = name;
    data['adminEmailAddress'] = adminEmailAddress;
    data['connectionString'] = connectionString;
    data['password'] = password;
    data['isActive'] = isActive;
    return data;
  }
}

File: beautify_api/8.0.0/aspnet-core/src/BanHangBeautify.Application/MultiTenancy/Dto/CreateTenantDto.cs
using Abp.AutoMapper;
using Abp.MultiTenancy;
using System;
using System.ComponentModel.DataAnnotations;

namespace BanHangBeautify.MultiTenancy.Dto
{
    [AutoMapTo(typeof(Tenant))]
    public class CreateTenantDto
    {
        [Required]
        [StringLength(16)]
        [RegularExpression(AbpTenantBase.TenancyNameRegex)]
        public string TenancyName { get; set; }

        [Required]
        [StringLength(AbpTenantBase.MaxNameLength)]
        public string Name { get; set; }

        //[Required]
        //[StringLength(AbpUserBase.MaxEmailAddressLength)]
        public string AdminEmailAddress { get; set; }

        [StringLength(AbpTenantBase.MaxConnectionStringLength)]
        public string ConnectionString { get; set; }
        public string Password { get; set; }

        public bool IsDefaultPassword { set; get; }
        public bool IsActive { get; set; }
        [Required]
        public int EditionId { get; set; }

        public bool IsTrial { get; set; }
        public DateTime? SubscriptionEndDate { get; set; }
    }
}
